api/typeDefs/notifications.js

```diff
@@ -156,11 +149,37 @@ export default gql`
     sortTime: Date!
   }
 
+  type CowboyHat {
+    id: ID!
+    sortTime: Date!
+    days: Int
+  }
+
+  type NewHorse {
+    id: ID!
+    sortTime: Date!
+  }
+
+  type LostHorse {
+    id: ID!
+    sortTime: Date!
+  }
+
+  type NewGun {
+    id: ID!
+    sortTime: Date!
+  }
+
+  type LostGun {
+    id: ID!
+    sortTime: Date!
+  }
```

Comment:
Instead of `type Horse` with a `found: Boolean` field, I decided to create dedicated types so I don't have to write code just to distinguish the types when I already can distinguish them I query for the rows.

I tried it first with `found: Boolean` but it sucked iirc.